content/development/coding-guidelines.md

```diff
@@ -78,6 +78,10 @@ Foo::Bar(const char* bar1, T bar2)
 {
 	...
 }
+
+//  Empty functions converted to single line functions
```

Comment:
Should this be allowed outside of the class declaration?

I think this is fine:

```
class Example
{
    void EmptyFunction() {}
};
```

But this isn't:

```
void Example::EmptyFunction() {}
```

In the case it's outside the class declaration it should instead be:

```
void
Example::EmptyFunction()
{
}
```

The case where an empty function is defined outside a class should be quite uncommon anyway. We would usually put it directly in the class declaration.